capa-spi-aws-log/src/main/java/group/rxcloud/capa/spi/aws/log/service/CloudWatchLogsService.java

```diff
@@ -148,7 +148,7 @@ private static void createLogGroup() {
                         .build();
                 CLOUD_WATCH_LOGS_CLIENT.createLogGroup(createLogGroupRequest);
             }
-        } catch (Exception e) {
+        } catch (Throwable e) {
```

Comment:
We already defined `CapaException`, the best way is defiend your exception extend CapaException.
Why use raw runtimeexception

Reply:
ok, I will fix it.